src/content/docs/setup/analytics/instrumentation.mdx

```diff
@@ -1,58 +1,205 @@
+The [Adobe Client Data Layer (ACDL)](https://github.com/adobe/adobe-client-data-layer) is a standardized JavaScript framework that simplifies data collection on your storefront. It provides a unified approach to capturing, storing, and transmitting user interaction data.
+
+### Key Capabilities
 
-- Collect data about what visitors experience on the web page
-- Communicate this data to digital analytics and reporting servers
+The ACDL enables your storefront to:
 
-The ACDL provides an API to:
+- **Collect interaction data**: Track user behaviors like product views, searches, cart actions, and purchases
+- **Standardize data format**: Ensure consistent data structure across all events
+- **Manage event timing**: Control when and how data is sent to analytics services
+- **Support multiple integrations**: Work seamlessly with Adobe Experience Platform, Analytics, and other tools
 
-- Register data that is to be merged into the data layer state.
-- Trigger events that relate to the data stored in the data layer.
-- Get the current data layer state of all merged data.
-- Register listeners that are called for specific events or data changes.
-- List the history of registered data and triggered events.
+### Core API Functions
 
-:::tip
-The Commerce boilerplate includes the ACDL by default.
-:::
+| Function | Description | Use Case |
+|----------|-------------|----------|
+| `push()` | Add data or trigger events | Send product view, cart addition events |
+| `getState()` | Retrieve current data layer state | Access user session or cart information |
+| `addEventListener()` | Register event listeners | React to specific user actions |
+| `getHistory()` | View event history | Debug or audit data collection |
+
+<Aside type="tip" title="Built-in Support">
+The Adobe Commerce boilerplate includes ACDL by default, so you don't need to install it separately. Drop-in components automatically send events to the data layer.
+</Aside>
 
 ## Configuration
 
-To configure variables for your Commerce backend, update the `analytics` object located in either your site config or a `config.json` in your repository. It should resemble the `analytics` property on https://aemshop.net/config.json.
+### Store Configuration
+
+To enable proper event collection, you need to configure your store's analytics settings. This configuration tells the instrumentation system about your store's identity and structure. Refer to the <Link href="https://experienceleague.adobe.com/developer/commerce/storefront/setup/configuration/commerce-configuration/" text="Commerce Configuration documentation" /> for more information.
+
+### Required Configuration Parameters
+
+<CodeExample code={`
+ ...
+  "analytics": {
+    "base-currency-code": "USD",
+    "environment": "Production",
+    "store-id": 1,
+    "store-name": "Main Website Store",
+    "store-url": "{{YOUR_STORE_URL}}",
```

Comment:
can we add value as `https://adobe-commerce.example.com` or something? as it looks like all other values are hardcoded and this one is configurable